datafusion/sqllogictest/test_files/copy.slt

```diff
@@ -390,11 +400,11 @@ query IT
 COPY source_table
 to 'test_files/scratch/copy/table_csv_with_options'
 (format csv,
-header false,
-compression 'uncompressed',
-datetime_format '%FT%H:%M:%S.%9f',
-delimiter ';',
-null_value 'NULLVAL');
+'csv.has_header' false,
+'csv.compression' uncompressed,
+'csv.datetime_format' '%FT%H:%M:%S.%9f',
+'csv.delimiter' ';',
+'csv.null_value' 'NULLVAL');
```

Comment:
From a UX perspective, I think the original was easier to write

Could we possible make both 

```sql
COPY source_table
to 'test_files/scratch/copy/table_csv_with_options'
(format csv,
header false,
compression 'uncompressed',
datetime_format '%FT%H:%M:%S.%9f',
delimiter ';',
null_value 'NULLVAL');
```

AND 

```sql
'csv.has_header' false,
'csv.compression' uncompressed,
'csv.datetime_format' '%FT%H:%M:%S.%9f',
'csv.delimiter' ';',
'csv.null_value' 'NULLVAL');
```

Accepted? Perhaps we could do so by first checking directly with the option name, and if that didn't find a matching config option, try `<format>.<config_name>` if the `format` option has been specified